Engine/insert.c

```diff
@@ -1881,14 +1881,14 @@ int subinstr(CSOUND *csound, SUBINST *p)
     return csoundPerfError(csound, &(p->h),
                            Str("subinstr: not initialised"));
   }
+  // This not needed since spout buffer is already managed by
```

Comment:
Looking at this comment, does it mean the code below should be removed?

Reply:
spurious comment left over from earlier changes. removed it